app/Controllers/TasksController.ts

```
@@ -0,0 +1,197 @@
+        .orderBy('isFavorite', 'desc') // Ordena os favoritos primeiro
+        .paginate(page, limit)
+
+      // Mapear os resultados para o formato desejado
+      const tasksJSON = tasks.toJSON()
+      tasksJSON.data = tasksJSON.data.map((task) => {
+        return {
+          id: task.id,
+          title: task.title,
+          description: task.description,
+          isCompleted: task.isCompleted ? true : false,
+          isFavorite: task.isFavorite ? true : false,
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)